pygitguardian/client.py

```diff
@@ -454,6 +455,30 @@ def multi_content_scan(
 
         return obj
 
+    def retrieve_secret_incident(
+        self, incident_id: int, with_occurrences: int = 0
+    ) -> Union[Detail, SecretIncident]:
+        """
+        retrieve_secret_incident handles the /incidents/secret/{incident_id} endpoint of the API
+
+        :param incident_id: incident id
+        :param with_occurrences: number of occurrences of the incident to retrieve (default 0)
```

Comment:
Can you clarify whether `0` means no occurrences are retrieved or if it means the  server uses its own default value?

Reply:
I had put 0 when I didn't think we wanted occurrences but I meant to change this default to 20 (which is the default of the API), done now.